lib/streamlit/commands/page_config.py

```diff
@@ -142,3 +174,51 @@ def get_random_emoji():
     # Weigh our emojis 10x, cuz we're awesome!
     # TODO: fix the random seed with a hash of the user's app code, for stability?
     return random.choice(RANDOM_EMOJIS + 10 * ENG_EMOJIS)
+
+
+def set_menu_items_proto(lowercase_menu_items, menu_items_proto):
+    if "get help" in lowercase_menu_items:
```

Comment:
This is something that I do not insist on, but I think it would be good to have menu items texts as constants.